scene/main/timer.cpp

```diff
@@ -206,6 +222,9 @@ void Timer::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_paused", "paused"), &Timer::set_paused);
 	ClassDB::bind_method(D_METHOD("is_paused"), &Timer::is_paused);
 
+	ClassDB::bind_method(D_METHOD("set_ignore_time_scale", "ignore"), &Timer::set_ignore_time_scale);
+	ClassDB::bind_method(D_METHOD("is_ignore_time_scale"), &Timer::is_ignore_time_scale);
```

Comment:
`is_ignore_time_scale()` isn't the greatest method name, from a "language grammar" perspective. It does match `SceneTreeTimer::is_ignore_time_scale()` (although, that method isn't exposed on `SceneTreeTimer`, so users can't see it) and the other nearby methods, so I think it could be OK

However, perhaps `get_ignore_time_scale()` would be better?

Reply:
I'd name it `is_ignoring_time_scale/set_ignire_time_scale`

Reply:
Thank you both. I named it that way to be consistent with `SceneTreeTimer`, but I do agree that it's not the best grammar. Since this one is going to be exposed, I renamed it as suggested.
I went with `get_ignore_time_scale()` as I feel it makes more sense to have something similar to the setter, and it makes grepping through the code more reliable.